### DISCUSSION TITLE
What happened to the list view (per day)?

### DISCUSSION THREAD

**Discussion by barseghyanartur** [psincraian/pepy#570] [Q&A]
What happened to the list view (per day)?

I only see graphs now, and I prefer to see the list.

**Comment by psincraian** [comment]
I thought no one was using it, so thanks for raising the issue :) I will add it back

**Reply by barseghyanartur** [reply]
Thanks! One suggestion I could come up with is perhaps to send out a survey for the future changes (for instance, using Google Forms). PePy had become a product, and it wouldn't be weird to ask what end users actually think.

**Reply by psincraian** [reply]
Now the table is back again :) thanks for raising and the suggestions
